cmd/ctl/pkg/convert/convert.go

```diff
@@ -159,7 +155,9 @@ func (o *Options) Complete() error {
 func (o *Options) Run() error {
 	builder := new(resource.Builder)
 
-	r := builder.Unstructured().LocalParam(true).ContinueOnError().
+	r := builder.
+		WithScheme(scheme, schema.GroupVersion{Group: cmapiv1alpha2.SchemeGroupVersion.Group, Version: runtime.APIVersionInternal}).
+		LocalParam(true).ContinueOnError().
```

Comment:
I think we *don't* want to `ContinueOnError` in this one, as if converting a single resource fails we should probably bail out immediately instead of silently continuing.